manifests/rgw.pp

```diff
@@ -98,8 +98,8 @@
     warning( 'The syslog parameter is unused and deprecated. It will be removed in a future release.' )
   }
 
-  unless $name =~ /^radosgw\..+/ {
-    fail("Define name must be started with 'radosgw.'")
+  unless $name =~ /^radosgw|rgw\..+/ {
```

Comment:
As discussed in Flowdock, `(radosgw|rgw)` would be a bit more robust and also the same style as in manifests/nodes.pp